audbcards/core/dataset.py

```diff
@@ -510,6 +512,22 @@ def tables(self) -> typing.List[str]:
         tables = list(db)
         return tables
 
+    @functools.cached_property
```

Comment:
The changes make sense, but it would break compatibility with the current cache. So, I would prefer to stay with the current solution.

Reply:
I think, I'm wrong on this one. I will double check.

Reply:
The thread that is unresolved afaics.  We will not need anything that handles caching at the level of a single property and no lazy loading.  The implementation constructs a dictionary comprehension that iterates over self._tables_stats, extracting the number of columns for each table.  As `_tables_stats` is a cached property itself, it will only be triggered once. The `_cached_properties` method only extracts properties from non-private properties and those that are not excluded via the `_table_related_cached_properties`.  For me pytest -s -v   --color=yes 'tests/test_dataset.py::TestDatasetLoadTables::test_load_tables' also passes.

Reply:
It is compatible with existing caches, but it would be inconsistent with our current approach. I propose to tackle this approach more holistic in another pull request and created https://github.com/audeering/audbcards/issues/115 to track it.

Reply:
In other words you are going to adapt the two-fold structure that has "private" cached properties and "public" non-cached ones a step further. 

So I could approve here, proceeding with this in #115 ?

Reply:
Yes, I think it make sense to tackle this issue more broadly in the way you described, but not in this pull request.

So, you can approve here if you are fine with the other changes.

Reply:
So I will resolve this thread, and write a short note in an approving review.